<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddForm</class>
 <widget class="QWidget" name="AddForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить задание</string>
  </property>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>86</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить!</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time_input">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="task_input">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Что нужно сделать?</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_input">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>97</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Время</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Заметки</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="note_input">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Место</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="place_input">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
